src/Uno.UI/UI/Xaml/Media/ImageSource.cs

```diff
@@ -54,8 +67,10 @@ private void InitializeDownloader()
 		{
 			Downloader = DefaultDownloader;
 		}
-
-#if !(__NETSTD__)
+ 
+#if (__NETSTD__)
+		protected IRandomAccessStream _stream;
```

Comment:
This is moved from `BitmapSource` to here, but I don't see any new usage of it. Can't it be in `BitmapSource` as previously?

Reply:
Well an `ImageSource` should not have any backing field for the sub-classes implementation (stream uri, or anything else). Some where added in early versions of uno and we are trying to move them away. We must not add a new one for `_NETSTD_`, so please remove it.

Reply:
Ok!